common/services/prismic/events.js

```diff
@@ -400,14 +401,13 @@ export function filterEventsForWeekend(events: UiEvent[]): UiEvent[] {
 }
 
 export function orderEventsByNextAvailableDate(events: UiEvent[]): UiEvent[] {
-  const reorderedEvents = [...events].filter(getNextDateInFuture).sort((a, b) => {
-    const aNextDate = getNextDateInFuture(a);
-    const bNextDate = getNextDateInFuture(b);
-
-    return aNextDate && bNextDate ? london(aNextDate.range.startDateTime)
-      .isBefore(bNextDate.range.startDateTime) ? -1  : 1 : -1;
+  const reorderedEvents = sortBy([...events].filter(getNextDateInFuture), event => {
+    const nextDateInFuture = getNextDateInFuture(event);
+    return nextDateInFuture;
```

Comment:
JavaScript question: is there a significance to declaring `nextDateInFuture` as a `const` rather than returning it directly? Or is it just a style choice?